client/src/modules/stock/StockEntryExitType.service.js

```diff
@@ -0,0 +1,100 @@
+angular.module('bhima.services')
+.service('StockEntryExitTypeService', StockEntryExitTypeService);
+
+StockEntryExitTypeService.$inject = [];
+
+function StockEntryExitTypeService() {
+  var $ctrl = this;
+
+  $ctrl.entryExitTypeList = [
```

Comment:
Do you need to bind this to the service?  It actually looks like you could keep this list a private variable, and only expose it through the "getAllowedTypes()" api.